docs/apache-airflow-providers-microsoft-azure/connections/azure_container_volume.rst

```diff
@@ -61,9 +63,9 @@ Extra (optional)
     Specify the extra parameters (as json dictionary) that can be used in Azure connection.
     The following parameters are all optional:
 
-    * ``connection_string``: Connection string for use with connection string authentication.
-    * ``subscription_id``: The ID of the subscription used for the initial connection. This is needed for Azure Active Directory (Azure AD) authentication.
-    * ``resource_group``: Azure Resource Group Name under which the desired Azure file volume resides. This is needed for Azure Active Directory (Azure AD) authentication.
+    * ``connection_string``: Connection string for use with connection string authentication.     It can be left out to fall back on DefaultAzureCredential_.
```

Comment:
```suggestion
    * ``connection_string``: Connection string for use with connection string authentication. It can be left out to fall back on DefaultAzureCredential_.
```